BUILD.gn

```diff
@@ -331,6 +331,7 @@ source_set("electron_lib") {
     "//components/network_session_configurator/common",
     "//components/pref_registry",
     "//components/prefs",
+    "//components/strings:components_strings_grit",
```

Comment:
I think these should be added to `electron_paks`, not directly here

Reply:
In Chromium this target is a dep pf targets that include "components/strings/grit/components_strings.h" (example: //components/printing/browser) and it seems corrects as the header is an output of that target:
`grit("components_strings") {
  source = "../components_strings.grd"
  defines = [
    "enable_arcore=$enable_arcore",
    "enable_plugins=$enable_plugins",
    "enable_print_preview=$enable_print_preview",
    "enable_vr=$enable_vr",
  ]

  outputs = [ "grit/components_strings.h" ]
  foreach(locale, locales_with_pseudolocales) {
    outputs += [ "components_strings_$locale.pak" ]
  }

  if (is_android) {
    outputs += android_components_strings_java_resources
  }
}`

At the same time "//components/strings:components_strings" is a dep of repack_locales("${target_name}_locales") in //electron/electron_packs.gni, so I believe we're on the safe side here.